app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -157,10 +160,16 @@ class Core(private val context: Context) {
      * @return the constructed tracking protection policy based on preferences.
      */
     fun createTrackingProtectionPolicy(
-        normalMode: Boolean = Settings.getInstance(context).shouldUseTrackingProtection,
+        normalMode: Boolean = Settings.getInstance(context).shouldUseTrackingProtectionGlobal,
         privateMode: Boolean = true
     ): TrackingProtectionPolicy {
-        val trackingProtectionPolicy = TrackingProtectionPolicy.recommended()
+        val trackingProtectionPolicy =
+            if (Settings.getInstance(context).shouldUseStrictTP) TrackingProtectionPolicy.all() else
+                TrackingProtectionPolicy.select(
+                    TrackingProtectionPolicy.RECOMMENDED - TrackingProtectionPolicy.SOCIAL +
```

Comment:
I think we want to have `SOCIAL` in the **standard option**
<img width="374" alt="Screen Shot 2019-07-26 at 10 31 14 AM" src="https://user-images.githubusercontent.com/773158/61959047-8a2f0700-af90-11e9-950f-a7736b845095.png">

Reply:
We have decided to take social out of standard until we get the new STP list because of too much breakage with the current list #3571 #4277

Reply:
Ok :) 👍